x-pack/platform/test/tsconfig.json

```diff
@@ -139,6 +139,7 @@
     "@kbn/gen-ai-functional-testing",
     "@kbn/sse-utils-server",
     "@kbn/inference-common",
-    "@kbn/rule-registry-plugin"
+    "@kbn/rule-registry-plugin",
+    "@kbn/security-solution-plugin"
```

Comment:
There is a strict requirement to avoid imports from solutions code in platform, including `x-pack/platform/test`: `security-solution-plugin` is a private SecSolution plugin and if it is required by test helper or tests, this code is considered a solution-specific scenario and should be placed in `x-pack/solutions/security/test` dir.

Reply:
It was an import of type, [removed](https://github.com/elastic/kibana/pull/222032/commits/086741092df46cf004a1abe8feec5b5c3a325f5e#diff-451e7daa261cedf8349091af588e4b777b4a4608a82f88f557e5ffd4aecb0b69)